nginx-php-moodle/Dockerfile

```diff
@@ -109,6 +109,22 @@ RUN echo "Starting Moodle eMailTest install..." \
     && find /opt/moodle/app/local/mailtest/ -type f -exec chmod 644 {} \; \
     && rm -fv /tmp/mailtest.zip
 
+RUN echo "Starting Moodle Custom certificate module install..." \
+    && curl -L -o /tmp/customcert.zip https://moodle.org/plugins/download.php/17375/mod_customcert_moodle35_2018051704.zip \
```

Comment:
Should this path be hard coded or added as a variable somewhere?

Reply:
It would only be in this file - I tried to make variables but the number in the path also changes along with the filename so the variables is basically a full URL

Reply:
OK.